articles/extensions/deploy-cli/references/deploy-cli-options.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Deploy CLI Options
+description: Describes the Auth0 Deploy CLI tool options.
+topics:
+  - extensions
+  - deploy-cli
+contentType:
+  - index
+  - concept
+useCase: extensibility-extensions
+---
+# Deploy CLI Options
+
+The following options are supported by the Deploy CLI tool `a0deploy`.
+
+## Commands
+
+- `a0deploy import` Deploy Configuration
+- `a0deploy export` Export Auth0 Tenant Configuration
+
+## Options
+- `--help` Show help  [boolean]
+- `--version` Show version number  [boolean]
+- `--verbose, -v` Dump extra debug information.  `[string] [default: false]`
```

Comment:
inconsistent formatting. "[boolean]" above is not monospace, but `[string] [default: false]` and `[string]` on the line below are